hack/tests/alpha-olm-subcommands.sh

```diff
@@ -23,18 +23,12 @@ test_version() {
     echo $commandoutput | grep -F "Failed to install OLM version \\\"${version}\\\": detected existing OLM resources: OLM must be completely uninstalled before installation"
 
     # Status should succeed with OLM installed
-    # If version is "latest", also run without --version flag
-    if [[ "$version" == "latest" ]]; then
-        commandoutput=$(operator-sdk alpha olm status 2>&1)
-        echo $commandoutput | grep -F "Successfully got OLM status for version \\\"${version}\\\""
```

Comment:
The primary purpose of this test is to make sure that `"latest"` is used as the default version when `--version` is not specified. Maybe we should move this logic to the `install` subcommand, and use `operator-sdk alpha olm install` when `$version == "latest"`?